doc/Integration.rst

```diff
@@ -0,0 +1,67 @@
+Integration
+===========
+
+Doctrine integration
+--------------------
+
+Independently from your preferred framework's characteristics, it is also possible to use plain
+Doctrine with the Money VO, using its Embeddables_ feature.
+
+.. _Embeddables: http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+
+Since you should not use annotations on the third party's library files, you'll use the `YAML mapping`_.
```

Comment:
yeah actually that's a leftover from a rephrase; was meant rather like

> Since [...] you'll want to use [...].

How about that?